### PR TITLE
Save / Load stuff

### PR DESCRIPTION
**Pull request by mennoo1996**
There is now a prompt on close when there are unsaved changes
Also, on start, the most recent project is automatically loaded

### CODE REVIEW

**Comment by Markvanderuit** [comment]
Looks good. Second member can merge.

File: src/main/java/gui/modal/SaveModalView.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    private Button saveButton;
+    
+    @Getter
+    private Button dontSaveButton;
+    
+    @Getter
+    private Button cancelButton;
+    
+    private VBox viewPane;
+
```
Comment (bart-de-jonge) [inline src/main/java/gui/modal/SaveModalView.java:27]: whitespace

File: src/main/java/gui/root/RootPane.java [hunk 2]
```diff
@@ -31,19 +38,17 @@
     private RootCenterArea rootCenterArea;
     @Getter
     private ControllerManager controllerManager;
+
```
Comment (bart-de-jonge) [inline src/main/java/gui/root/RootPane.java:41]: whitespace

File: src/main/java/gui/root/RootPane.java [hunk 3]
```diff
@@ -18,6 +22,9 @@
     private int minimumResolutionY = 480;
     private int startingResolutionX = 800;
     private int startingResolutionY = 600;
+    
+    @Getter
+    private static final String CONFIG_FILEPATH = "config.txt";
```
Comment (bart-de-jonge) [inline src/main/java/gui/root/RootPane.java]: change to .ini

**Comment by bart-de-jonge** [comment]
Great work

**Pull request merged**